packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/services/publicCloud.ts

```diff
@@ -412,9 +412,21 @@ const pciNode: Node = {
       id: 'pci-ai',
       idAttr: 'pci-ai-link',
       translation: 'sidebar_pci_ai',
-      features: ['notebooks', 'ai-apps', 'training'],
+      features: ['notebooks', 'ai-apps', 'training', 'ai-dashboard'],
       forceVisibility: true,
       children: [
+        {
+          id: 'pci-ai-dashboard',
+          idAttr: 'pci-ai-dashboard-link',
+          translation: 'sidebar_pci_ai_dashboard',
+          serviceType: 'CLOUD_PROJECT_INSTANCE',
```

Comment:
Isn't there a specific serviceType for AI Dashboard ?

Reply:
well seen, I had not seen this property, I just corrected.